ambry-api/src/main/java/com.github.ambry/account/Container.java

```diff
@@ -63,6 +63,7 @@
   static final String STATUS_KEY = "status";
   static final String DESCRIPTION_KEY = "description";
   static final String IS_PRIVATE_KEY = "isPrivate";
+  static final String IS_BACKED_UP_KEY = "isBackedUp";
```

Comment:
Does this field mean that blobs in this container are required to be backed up in cloud? If so, maybe we could rename it to `backUpRequired` or `cloudBackUpRequired`?

Reply:
I vote for `backupEnabled`.

Reply:
will change to backupEnabled